### ISSUE TITLE
Changing image_size to 256 triggers error

### ISSUE THREAD

**Issue by Cristy94** [bamos/dcgan-completion.tensorflow#56]
I can't seem to be able to change the image size or number of initial filter values, if I do that I get this error:

` F C:\tf_jenkins\home\workspace\rel-win\M\windows-gpu\PY\35\tensorflow\stream_executor\cuda\cuda_dnn.cc:430] could not convert BatchDescriptor {count: 4 feature_map_count: 0 spatial: 129 129  value_min: 0.000000 value_max: 0.000000 layout: BatchDepthYX} to cudnn tensor descriptor: CUDNN_STATUS_BAD_PARAM`

**Comment by sachinjm** [comment]
I also faced same problem while running for image of size 128

**Comment by DeliaX** [comment]
@Cristy94  I have also come across with this issue when training a CNN-based model, so I wonder whether you have solved this problem or not. If yes, could you please share your solution? Thank you!

**Comment by Cristy94** [comment]
@DeliaX No, I haven't found a solution, maybe someone has a fork where using a different image size works.

**Comment by AriesSurfer** [comment]
if the targeted size (200*200) > the original size(128*128) ,it will raise this error.  just filter to process this small size image

**Comment by pavibalaji** [comment]
@Cristy94 I am also looking for 256 x 265 size image solutions. Do you have any lead on this? Thanks.

**Comment by Cristy94** [comment]
@pavibalaji  No, haven't solved the issue.

**Comment by Aurevious** [comment]
Hi，have anyone solved the issue?
